scripts/release-dockerhub.sh

```diff
@@ -43,10 +43,18 @@ function getTagsArg() {
   done
 }
 
+function dockerBuildxSetup() {
+  echo Setting Up Docker Buildx
+  echo ========================
+  docker buildx create --platform=linux/arm64,linux/amd64 --use
+  echo ========================
+  echo ✅ Build completed ${2} from ${1} 
+}
+
 function dockerBuild() {
   echo 🔨 Building ${2} from ${1}: docker build --file ${1}/Dockerfile $(getTagsArg ${2})
   echo ========================
-  docker build --file ${1}/Dockerfile $(getTagsArg ${2}) .
+  docker buildx --file ${1}/Dockerfile --platform=linux/arm64,linux/amd64 $(getTagsArg ${2}) .
```

Comment:
Using `docker buildx build ...` and afterwards `docker push ...` will not work. The images + manifest for multiarch docker images are not added to your local docker registry but are only "temporary" in the buildx environment. Instead you can use [`--push`](https://docs.docker.com/engine/reference/commandline/buildx_build/#push) to directly build+push and remove the `dockerPush` step.

Also note that due to the new feature `SLSA Provenance attestation`, which is enabled by default since January's 0.10.x release of `docker buildx` (see [release notes](https://github.com/docker/buildx/releases/tag/v0.10.0)), the produced image manifest will be of media type `application/vnd.docker.distribution.manifest.list.v1+json` instead of `application/vnd.docker.distribution.manifest.list.v2+json`. This causes problems on some CI platforms, such as Google Cloud or Jfrog artifactory. I would recommend using `--provenance=false` for compatibility reasons.

Reply:
You are right, indeed. The way you have described it, is how we do it for our private repos, but was not sure of this project's structure so I did not make any more changes to the push of the Docker image. I will update the PR with your comments 🙏🏻

Reply:
@dlavrenuek PR updated!